documental/serializers.py

```diff
@@ -24,9 +25,9 @@ class UsuarioSerializers(serializers.ModelSerializer):
 
     class Meta:
         """Meta class for `UsuarioSerializers` serializer."""
```

Comment:
Put blank line after docstrings.

Reply:
Trailing whitespaces.

Use pep8 to format this file.

Reply:
.